shared/src/androidMain/kotlin/com/macaosoftware/sdui/app/Platform.kt

```diff
@@ -0,0 +1,14 @@
+package com.macaosoftware.sdui.app
```

Comment:
Remove this file, it do nothing in this PR

Reply:
This is the common file in all the platform. I just created this to remember that it's the platform. In this we will add expect and actual functions